shell/browser/web_view_guest_delegate.cc

```diff
@@ -105,13 +105,18 @@ content::WebContents* WebViewGuestDelegate::CreateNewGuestWindow(
   guest_params.context = embedder_web_contents_->GetNativeView();
   std::unique_ptr<content::WebContents> guest_contents =
       content::WebContents::Create(guest_params);
-  content::RenderWidgetHost* render_widget_host =
-      guest_contents->GetRenderViewHost()->GetWidget();
-  auto* guest_contents_impl =
-      static_cast<content::WebContentsImpl*>(guest_contents.release());
-  guest_contents_impl->GetView()->CreateViewForWidget(render_widget_host);
-
-  return guest_contents_impl;
+  if (!create_params.opener_suppressed) {
+    auto* guest_contents_impl =
+        static_cast<content::WebContentsImpl*>(guest_contents.release());
+    auto* new_guest_view = guest_contents_impl->GetView();
+    content::RenderWidgetHostView* widget_view =
+        new_guest_view->CreateViewForWidget(
+            guest_contents_impl->GetRenderViewHost()->GetWidget());
+    if (!create_params.initially_hidden)
+      widget_view->Show();
```

Comment:
this call to `Show` is new, is it needed? Is this code inspired by some analogous code in Chromium?

Reply:
It is analogous to the non-guest code path from `content::WebContentsImpl::CreateNewWindow`, this one specifically to https://source.chromium.org/chromium/chromium/src/+/main:content/browser/web_contents/web_contents_impl.cc;l=3794-3798